### ISSUE TITLE
Wrong error message when the auth. name servers of a zone are all down

### ISSUE THREAD

**Issue by bortzmeyer** [zonemaster/zonemaster#1165]
Yesterday, the name servers for `ordipat.fr`  were all down.

But the error message is wrong:

```
Aucun enregistrement de type "NS" renvoyé par la zone parente pour la zone testée. Les tests complémentaires sur ces serveurs sont annulés. 
```

The parent zone (`.fr`) worked fine at this time, it was the child zone which had a problem.

https://zonemaster.net/fr/result/7dc9b7d47349de09

**Comment by matsduf** [comment]
Presently, Zonemaster does not correctly report some types of complete unavailability. That will be fixed in the upcoming version.

**Comment by matsduf** [comment]
@bortzmeyer With the new version (v2023.1) you can see a difference between tested domain (zone) does not exist (https://zonemaster.net/en/result/0bd514f05c49360e) versus tested domain (zone) is delegated but all of the nameservers are lame (https://zonemaster.net/en/result/cde55bc8d6ffe728).

Does this resolv the issue for you?

**Comment by bortzmeyer** [comment]
Yes, it's fine.

**Comment by matsduf** [comment]
Resolved by v2023.1.

**Comment by matsduf** [comment]
Resolved by v2023.1.
